images/linux/Ubuntu1804-README.md

```diff
@@ -71,10 +70,10 @@
 - Git-ftp 1.3.1
 - Haveged 1.9.1
 - Heroku 7.56.1
-- HHVM (HipHop VM) 4.118.2
+- HHVM (HipHop VM) 4.120.0
 - jq 1.5
 - Kind 0.11.1
-- Kubectl 1.20.1-5-g76a04fc
+- Kubectl 0.21.0-beta.1
```

Comment:
@dibir-magomedsaygitov is it expected? Do we want to install beta versions?

Reply:
Task for investigation
https://github.com/actions/virtual-environments-internal/issues/2524